first session/assignment/shoppingmall.css

```
@@ -0,0 +1,126 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.hamburger-menu .menu {
+    display: none;
+}
+
+
+.hamburger-icon {
+    background: none;
+    border: none;
+    font-size: 24px;
+    cursor: pointer;
+    padding: 10px;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+    position: fixed;
```

fixed 속성은 항상 뷰 어딘가에 고정되어있는 속성이죠!! 보통 header같이 항상 보여야 하는 요소들에 적용하는데 항상 보여야하다보니 다른 요소들에 가릴 수 있는 위험을 제거해줘야 해요! 이때 z-index라는 속성을 사용하는데, z-index는 z축 방향으로 순서를 결정해주는 속성이에요!! z축으로 요소를 올려서 항상 가려지지 않도록 하는거죠! 그래서 fixed 속성과 z-index속성을 동시에 쓰는 경우가 아주 많습니다!!!
참고로 z-index값을 너무 큰 값으로 설정하면 나중에 z축 순서가 꼬일 위험이 있어서 1부터 차례로 쌓아나가는것이 좋습니다!!

```suggestion
    position: fixed;
    z-index: 1;
```